Source/Core/VideoCommon/AVIDump.cpp

```diff
@@ -323,6 +323,11 @@ void AVIDump::CloseVideoFile()
   av_frame_free(&s_scaled_frame);
 
   avcodec_free_context(&s_codec_context);
+
+  if (s_format_context->pb)
```

Comment:
[avio_close](https://github.com/FFmpeg/FFmpeg/blob/5fe2b437023f46394dfd4a4c991aa4a3ac3d8e72/libavformat/aviobuf.c#L1091) (which [avio_closep](https://github.com/FFmpeg/FFmpeg/blob/5fe2b437023f46394dfd4a4c991aa4a3ac3d8e72/libavformat/aviobuf.c#L1111) calls internally, already performs a null check on the data being passed to it).